doc/usermanual/darkroom/modules/effect/watermark.xml

```diff
@@ -374,7 +374,7 @@
                 $(Xmp.dc.description)
               </entry>
               <entry>
-                The description of the image
+                The description of the image.  This is the field to use when you're looking for the user-specified text that some cameras add at shoot time to exif field "User Comment" (also called "Exif.Photo.UserComment").
```

Comment:
For consistency with the rest of the manual please replace starting/leading double quotes with `<quote></quote>`.

Reply:
Done, see new commit. Thanks Ulrich for notifying. Future contributions will have correct quote mechanism without asking.

Reply:
Done, see new commit. Thanks Ulrich for notifying. Future contributions will have correct quote mechanism without asking.